lib/systems/platforms.nix

```diff
@@ -255,6 +255,7 @@ rec {
       arch = "armv6";
       fpu = "vfp";
       float = "hard";
+      abi = "aapcs-linux";
```

Comment:
I'm going to add that commented out, for someone else that actually knows about raspberry pis to triage.